src/test.js

```diff
@@ -374,7 +374,9 @@ Test.prototype = {
 				resume = internalStop( test );
 				then.call(
 					promise,
-					function() { resume(); },
+					function() {
+						resume();
+					},
```

Comment:
Turns out it was me after all (I thought there was a violation earlier but I must have tweaked a config to make that okay). I'll revert this.